.github/workflows/test.yml

```diff
@@ -3,12 +3,12 @@ name: Test
 on:
```

Comment:
I think if you remove submodules you should also modify workflows/assets.yml to remove the submodules: recursive line

Reply:
Good spot. I had left that `yml` file untouched (and unchecked).